steward/testfile.toml

```diff
@@ -0,0 +1,5 @@
+keystore = "mytest"
```

Comment:
I don't think this file is used anywhere, we can probably delete it.

Reply:
Removed it!